scripts/ci/5-run-tests.sh

```diff
@@ -94,5 +94,11 @@ fi
 echo "Starting the unit tests with the following nose arguments: "$nose_args
 nosetests $nose_args
 
+# Fix codecov build path
+if [ ! -h /home/travis/build/apache/incubator-airflow ]; then
+  sudo mkdir -p /home/travis/build/apache
+  sudo ln -s ${ROOTDIR} /home/travis/build/apache/incubator-airflow
```

Comment:
Codecov uses Travis envs to get the build path but because the tests are running in the container the path is `/app` instead of `/home/travis/build/apache/incubator-airflow`

Codecov logs:
```
==> Preparing upload
/bin/sh: 1: cd: can't cd to /home/travis/build/apache/incubator-airflow
    Error running `cd /home/travis/build/apache/incubator-airflow && git ls-files`: None
==> Processing gcov (disable by -X gcov)
    Executing gcov (find /home/travis/build/apache/incubator-airflow -not -path './bower_components/**' -not -path './node_modules/**' -not -path './vendor/**' -type f -name '*.gcno'  -exec gcov -pb  {} +)
find: ‘/home/travis/build/apache/incubator-airflow’: No such file or directory
    Error running `find /home/travis/build/apache/incubator-airflow -not -path './bower_components/**' -not -path './node_modules/**' -not -path './vendor/**' -type f -name '*.gcno'  -exec gcov -pb  {} +`: None
```